docs/source/community/persons_of_interest.rst

```diff
@@ -177,10 +210,15 @@ Model Compression & Optimization
 Windows
 ~~~~~~~
 
--  Peter Johnson (`peterjc123 <https://github.com/peterjc123>`__)
 -  Guoliang Hua (`nbcsm <https://github.com/nbcsm>`__)
 -  Teng Gao (`smartcat2010 <https://github.com/smartcat2010>`__)
```

Comment:
This profile https://github.com/smartcat2010 is 404. Should we remove it?

Reply:
@b0noI can you check? Thanks

Reply:
https://github.com/gaoteng-git this is the right one, however since there were no commits from him for almost a year I would suggest to remove him

Reply:
Please don't remove, but rather move to `(emeritus)`. That's our model.